tracer_test.go

```diff
@@ -42,3 +43,18 @@ func TestTracer_StartSpan_SuppressTracing(t *testing.T) {
 	sc := sp.Context().(instana.SpanContext)
 	assert.True(t, sc.Suppressed)
 }
+
+func TestTracer_StartSpan_WithCorrelationData(t *testing.T) {
+	recorder := instana.NewTestRecorder()
+	tracer := instana.NewTracerWithEverything(&instana.Options{}, recorder)
+
+	sp := tracer.StartSpan("test", ot.ChildOf(instana.SpanContext{
```

Comment:
Is this a valid use case? I didn't know that...

Reply:
`ot.ChildOf()` takes a span context, which is an interface type, so there it's possible to use any type that satisfies `ot.SpanContext` there, although you probably should not create your own parent contexts in production code... Here is just for testing purposes only.